### ISSUE TITLE
Confused about the data_normaliser steps.

### ISSUE THREAD

**Issue by jt827859032** [AI4Finance-LLC/FinRL-Library#3]
Thanks for your brilliant works. I'm curious about the <b>data_normaliser</b> code in <b>FinRL_single_stock_trading.ipynb, chapter 5.1. </b>
```
from sklearn import preprocessing
data_normaliser = preprocessing.StandardScaler()
train[feaures_list] = data_normaliser.fit_transform(train[feaures_list])
trade[feaures_list] = data_normaliser.fit_transform(trade[feaures_list])
```
The code seem to use future information for normalization, am I right?
